docs/Platform-Architecture/permissions-model.md

```diff
@@ -0,0 +1,159 @@
+| Delivery Team Admin | Delivery Project | Tech lead and/or Delivery Manager for the Delivery Project team. |
+| Delivery Programme Admin | Delivery Programme | Administers Delivery Programmes within the ADP Portal. |
+| ADP Admin | Platform | ADP Platform Engineering delivery team member. |
+| CCoE Engineer | Organization | Cloud Center of Excellence engineer. |
+
+!!! info
+    **Please note:** if a user holds multiple roles, they will be granted the permissions associated with their highest role. This ensures that they have all the access rights and privileges of the most significant role they possess.
+
+## Portal Permissions
+
+The permissions for the portal are stored both in a database and in Azure AD with the use of AAD groups. The groups are named as follows:
+
+- Delivery Team Member are assigned to `Delivery AAG-Users-ADP-{programme}-{devlivey project}_TeamMember` AAD group.
+- Technical Team Member are assigned to `AAG-Users-ADP-{programme}-{devlivey project}_TechUser` AAD group.
+- Delivery Team Admin are assigned to `AAG-Users-ADP-{programme}-{devlivey project}_Admin` AAD group.
+- Delivery Programme Admin are assigned to `AAG-Users-ADP-{programme}_Admin` AAD group.
+- ADP Admins are assigned to `AAG-User-ADP-PlatformEngineers` AAD group.
+
+By being added to these groups in Azure AD via the ADP Portal, users will be granted the permissions for their role in the ADP Portal.
+
+The permissions for each role in the ADP Portal are detailed below.
+
+### Platform User
+
+ADP Portal Permissions for the Platform User role:
+
+- Access to the ADP Portal.
+- Can be selected as a Delivery Project team member/admin or Delivery Programme Admin.
+- Read access to all ALBs, delivery projects, programmes, etc.
+
+### Delivery Project: Team Member
+
+ADP Portal Permissions for the Delivery Project Team Member role:
+
+- Includes all Platform User permissions.
+- Displayed as a Member of assigned Delivery Project teams.
+
+### Delivery Project: Technical Team Member
+
+ADP Portal Permissions for the Technical Team Member role:
+
+- Includes all Delivery Project Team Member permissions.
+- Scaffold/create new services for their delivery project (inc. repos).
+
+### Delivery Project: Team Admin
+
+ADP Portal Permissions for the Delivery Team Admin role:
+
+- Includes all Delivery Team Member permissions.
+- Has the ability to invite/add users to their Delivery Project team as: Team members, Technical Team Members, Team Admins, and Technical Team Admins via the ADP Portal. This will also add them to the required GitHub team, Azure DevOps project, and Azure AAD groups required for Azure resource access.
+- Edit delivery project details in the ADP Portal.
+
+### Delivery Programme Admin
+
+ADP Portal Permissions for the Delivery Programme Admin role:
+
+- Includes all Delivery Team Admin permissions for all Delivery Projects in the programme.
+- Can create new Delivery Projects in the programme.
+- Can edit programme details in the ADP Portal.
```

Comment:
Can only edit programme details for programmes they are admins of, not all programmes